command.go

```diff
@@ -273,6 +275,12 @@ func parseEnvArgs(args []string) (Command, error) {
 		return nil, ErrNotEnoughArguments
 	}
 
+	if assume != "" && noSession != false {
+		return nil, errors.New("Refusing to output variables. Because --assume generates session credentials it cannot be combined with --no-session.")
+	} else if refresh != false && noSession != false {
```

Comment:
It looks like `refresh` isn't defined here, we probably just need to query it like we are `assume` and `noSession` above.